src/modules/mediaplayer/MpMprisInterface.cpp

```diff
@@ -83,8 +48,8 @@ int MpMprisInterface::detect(bool)
 }
 
 #define MPRIS_SIMPLE_CALL(__path, __action)                                           \
```

Comment:
identifiers with double `_` ar reserved for compiler's own needs, so this may randomly break

Reply:
The new commit removes the double __